py4j-web/contributing.rst

```diff
@@ -113,7 +113,6 @@ suite across the supported python versions.
   ./gradlew assemble
 
   # install test requirements
-  cd py4j-python
```

Comment:
I think we should change this as `cd py4j-python` and `requirements.txt` -> `requirements-test.txt`